src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -16,7 +16,14 @@ import javax.net.SocketFactory
 import javax.net.ssl.SSLContext
 import org.broadinstitute.dsde.workbench.dataaccess.PubSubNotificationDAO
 import org.broadinstitute.dsde.workbench.google.GoogleCredentialModes.{Json, Pem}
-import org.broadinstitute.dsde.workbench.google.{GoogleDirectoryDAO, HttpGoogleDirectoryDAO, HttpGoogleIamDAO, HttpGoogleProjectDAO, HttpGooglePubSubDAO, HttpGoogleStorageDAO}
+import org.broadinstitute.dsde.workbench.google.{
+  GoogleDirectoryDAO,
```

Comment:
Out of curiosity, what causes each of these to show up on different lines? Just total line length? I think it is easier to read, but was just wondering about the inconsistency between this and line 29 as an example.